src/util/sss_ini.c

```diff
@@ -961,7 +961,7 @@ int sss_ini_read_sssd_conf(struct sss_ini *self,
         return EINVAL;
     }
 
-    ret = sss_ini_open(self, config_file, CONFDB_FALLBACK_CONFIG);
+    ret = sss_ini_open(self, config_file, "[sssd]\n");
```

Comment:
It creates empty context to add snippets.
That's what I was discussing: do we really want to support a case when main sssd.conf doesn't exist, but snippets exist?

Reply:
Hi,

it might be worth to explain the use-case of the fallback config in a comment. Additionally I's prefer to keep `CONFDB_FALLBACK_CONFIG` and change the `#define`.

bye,
Sumit

Reply:
Comment is sufficient for me, thanks. I'll wait for Sumit to Resolve conversation.

Reply:
Hi,

thanks for the comment.

bye,
Sumit